### DISCUSSION TITLE
How can i get the ResourceDictionary.Souce.OriginalString in ResourceDictionary.MergedDictoinaries

### DISCUSSION THREAD

**Discussion by BonanzaBegin** [AvaloniaUI/Avalonia#12118] [Q&A] [answered]
<Application.Resources>
        <ResourceDictionary>
            <ResourceDictionary.MergedDictionaries>
                <ResourceInclude Source="avares://AvaloniaPreview2/Culture/en-US.axaml"></ResourceInclude>
                <ResourceInclude Source="avares://AvaloniaPreview2/Culture/zh-CN.axaml"></ResourceInclude>
            </ResourceDictionary.MergedDictionaries>
        </ResourceDictionary>
    </Application.Resources>

I have added these ResouceDictionaries to appsource add use the code below to resolve OriginalString 

var resources = App.Current.Resources.MergedDictionaries.OfType<ResourceInclude>();
Console.WriteLine("*******************" + resources.Count());//2
foreach (var item in resources)
{
 //Avalonia 0.10.8 i can get the source.OriginalString
//Avalonia 11.0.0 it doesn't work  because the item in App.Current.Resources.MergedDictionaries is dictionary not 
 //ResourceInclude  so resources.Count==0;
Console.WriteLine(item.Source.OriginalString);
}

How can i get the ResourceDictionary.Souce.OriginalString in ResourceDictionary.MergedDictoinaries?

thanks

**Comment by maxkatz6** [comment, marked as answer]
You can't. It's replaced with a ResourceDictionary by the compiler.
Alternatively, you can put ResourceInclude in your resources with a Key and use it by a key later, or similar alternative solution.
